programs/protocol_event/src/lib.rs

```diff
@@ -79,4 +81,42 @@ pub mod protocol_event {
             name,
         )
     }
+
+    // Participant management instructions
+
+    pub fn create_individual_participant(
+        ctx: Context<CreateParticipant>,
+        code: String,
+        name: String,
+    ) -> Result<()> {
+        instructions::update_grouping::increment_category_participant_count(
+            &mut ctx.accounts.category,
+        )?;
```

Comment:
Intentionally starting the index from `1`?

Reply:
Yeah, I did have it start from 0 originally but in my head it is simpler to start with a team/player of id 1